migrations/migrations/20181126143436-create-user.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Users', {
+      id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      firstName: {
```

Comment:
By convention, we always name table colums in lower case and using snake case. In this case, this column should be named `first_name`. This applies to all other columns